pkg/driver/qemu/entitlementutil/entitlementutil.go

```diff
@@ -107,7 +107,5 @@ func AskToSignIfNotSignedProperly(ctx context.Context, qExe string) {
 		} else {
 			logrus.Warn("If QEMU does not start up, you may have to sign the QEMU binary with the \"com.apple.security.hypervisor\" entitlement manually. See https://github.com/lima-vm/lima/issues/1742 .")
 		}
-	} else {
-		logrus.Infof("QEMU binary %q seems properly signed with the \"com.apple.security.hypervisor\" entitlement", qExe)
```

Comment:
Should be harmless to keep this as a debug-level log

Reply:
> Should be harmless to keep this as a debug-level log

Yes, but why? How is this message ever useful?

I don't really mind keeping it (as debug), but I would like to understand the purpose of it.